dynamic_instrumentation/src/point/build.rs

```diff
@@ -150,7 +150,13 @@ impl<'tcx> InstrumentationBuilder<'_, 'tcx> {
                         func,
                         ..
                     } if destination.is_some() => {
-                        let mut s = format!("{:?} = {:?}(", destination.unwrap().0, func);
+                        let mut s = format!("{:?} = ", destination.unwrap().0);
+                        if let ty::FnDef(def_id, _) = func.ty(self.body, self.tcx).kind() {
+                            let name = self.tcx.item_name(*def_id);
+                            s.push_str(&format!("{}(", name));
+                        } else {
+                            s.push_str(&format!("{:?}(", func));
+                        };
```

Comment:
I hadn't realized this changed, which changes things like this:

```rust
std::vec::Vec::<*mut i8>::push(move _29, move _30);
```

to this:

```rust
push(move _29, move _30);
```

which is IMO a lot less readable.  Sorry, I just realized this on some other merge.

What we want is

```rust
Vec::<*mut i8>::push(move _29, move _30);
```

since `Vec` is in the `std` prelude.

If that's not easy, I think we should revert this change.  I'll open a PR to do so.